package.json

```diff
@@ -0,0 +1,9 @@
+{
+  "devDependencies": {
+    "bootstrap": "4.6.0",
```

Comment:
do we need to configure dependabot for this?

Reply:
done